src/reducers/lotesReducer.js

```diff
@@ -23,13 +23,22 @@ export default function loteReducer(state = initialState, action) {
           map(id => {
             const { title, artista, obras, price } = action.data[id];
 
+            let isWishlist = false;
```

Comment:
I don't think we need this bool anymore right? As there is another check in the header component.

Reply:
oh yea none of this should be here. isWishlist has nothing to do with lotesReducer

Reply:
maybe would've have made more sense to do the check here. but i didn't do it this way